src/ray/raylet/local_object_manager.cc

```diff
@@ -122,10 +122,10 @@ void LocalObjectManager::ReleaseFreedObject(const ObjectID &object_id) {
 
   RAY_LOG(DEBUG) << "Unpinning object " << object_id;
   // The object should be in one of these states: pinned, spilling, or spilled.
-  RAY_CHECK((pinned_objects_.count(object_id) > 0) ||
-            (spilled_objects_url_.count(object_id) > 0) ||
-            (objects_pending_spill_.count(object_id) > 0));
-  if (pinned_objects_.count(object_id)) {
+  RAY_CHECK(pinned_objects_.contains(object_id) ||
```

Comment:
nit: just a nit, I usually find having separate checks is more readable than combining them together.